manifests/repo/rhel.pp

```diff
@@ -6,18 +6,19 @@
   String $key_source = $rabbitmq::package_gpg_key,
 ) {
   yumrepo { 'rabbitmq':
-    ensure   => present,
-    name     => 'rabbitmq_rabbitmq-server',
-    baseurl  => $location,
-    gpgkey   => $key_source,
-    enabled  => 1,
-    gpgcheck => 1,
+    ensure        => present,
+    name          => 'rabbitmq_rabbitmq-server',
+    baseurl       => $location,
+    gpgkey        => $key_source,
+    enabled       => 1,
+    gpgcheck      => 0,
```

Comment:
are we disabling this because the package is no longer signed?

Reply:
@bishopbm1 while I see that the packagecloud builtin script does set `gpgcheck` to disabled, the RMQ docs do say the package itself should be signed.

https://www.rabbitmq.com/signatures.html
The bit under "Using a Key Server" mentions some other keys.
If at all possible, IMO it would be probably be best / safest to check both the signatures, even if it means we have to separately grab / import some additional keys.

Reply:
@wyardley I added the additional key that you referenced. It is working with both check enabled now.